ecosystem/typescript/sdk_v2/src/types/index.ts

```diff
@@ -42,6 +42,26 @@ export enum RustEnumTransactionVariants {
   FeePayerTransaction = 1,
 }
 
+/**
+ * Transaction Authenticator enum as they are represented in Rust
+ * {@link https://github.com/aptos-labs/aptos-core/blob/main/types/src/transaction/authenticator.rs#L44}
+ */
+export enum RustEnumTransactionAuthenticatorVariant {
+  TransactionAuthenticatorEd25519 = 0,
+  TransactionAuthenticatorMultiEd25519 = 1,
+  TransactionAuthenticatorMultiAgent = 2,
+  TransactionAuthenticatorFeePayer = 4,
```

Comment:
Those should be pretty easy to name simply.  Doesn't have to be that descriptive that it refers to rust.

TransactionAuthenticatorType

Then just 
Ed25519
MultiEd25519
MultiAgent
FeePayer

Reply:
I agree, though most of the time people look at it and ask "what are these numbers? where they come from?" it helped me a lot to figure out that these are fixed numbers that come from our rust code, but also the comment on each enum should help with that.

kk, let me change these names in a another PR

Reply:
yeah I feel that but I think we can just document that it comes from Rust, at some point we should be able to say "Read the documentation" rather than have to explain these things